about.html

```diff
@@ -0,0 +1,141 @@
+        </li>
+
+        <li class="list-items">
+          <a href="#">WITMP 2023</a>
+        </li>
+      </ul>
+    </nav>
+</header>
+<!-- Header Ends Here -->
+
+<!-- Main Starts Here -->
+<section class="about-theme-info">
+<!-- Hero -->
+<div class="about-first-section">
+<div>
+<h4 class="about-theme-intro">"Join us this year!"</h4>
+<h2 class="about-theme">WOMEN IN TECH <br> MENTORSHIP PROGRAMME <br> 2023.</h2>
+</div>
+
+<div class="about-intro">
+<h4 class="about-intro-text">
+Are you a woman trying to navigate the tech space? This mentorship program for women in tech contribute to the retention and advancement of female talent, as mentees gain confidence, expand their professional networks, and acquire the necessary skills to excel in their careers, leading to increased diversity and innovation in the tech sector. <span class="button-span">Click the button below to register now.</span>
+</h4>
+<a href="#" class="button-a">
+  <button class="button"> Click Here to Register  </button></a>
+</div>
+</div>
+
+<div class="about-second-section">
+<div class="about-program">
+  <h4 class="about-main-program">See past Women In Tech mentorship program</h4>
+  <hr class="about-one">
+  <h5 class="about-summit">Take a look at the past three Women In Tech mentorship program which took place in Abuja, in Lagos and in Enugu.</h5>
+</div>
+
+<div class="about-program-container">
+
+  <div class="about-program-details">
+    <h5 class="about-detail-name">2020</h5>
+    <p class="about-detail-text">Women In Tech Annual Mentorship program in Abuja.</p>
+  </div>
+
+  <div class="about-program-details1">
+    <h5 class="about-detail-name">2021</h5>
+    <p class="about-detail-text">Women In Tech Annual Mentorship program in Lagos.</p>
+  </div>
+
+  <div class="about-program-details2">
+    <h5 class="about-detail-name">2022</h5>
+    <p class="about-detail-text">Women In Tech Annual Mentorship program in Enugu.</p> 
+  </div>
+</div>
+
+
+<div class="about-third-section">
+<div class="about-detail-div">
+  <h4 class="about-detail-title">Please contact us per Email for any further questions about CC Global Summit 2015!</h4>
+<strong class="strong">witmprogram@gmail.com</strong>
+</div>
+</div>
+<!-- 
+<div>Testimonials??</div> -->
+
+<!-- Footer -->
+<footer class="about-footer">
+  <div class="about-footer-one">
+    <h6 class="about-partner">Partner</h6>
+    <hr class="about-one">
+    <div class="about-partner-img">
+      <img src="Assets\googlekorea 1.png" alt="google" class="about-partner-icon"/>
+      <img src="Assets\mozilla 1.png" alt="mozilla" class="about-partner-icon"/>
+      <img src="Assets\naver 1.png" alt="naver" class="about-partner-icon"/>
+      <img src="Assets\airbnb 1.png" alt="airbnb" class="about-partner-icon"/>
+    </div>
+  </div>
+  <div class="about-footer-two">
+    <div class="about-wit">
+      <h2 class="about-wit">MITMP</h2>
+    </div>
+    <div class="about-wit1">
```

Comment:
- [ ] great job. Please note that the footer section of the about page has a dark background
- [ ] kindly use just two images like in the original design and that section has a white background color. please kindly follow that.
- [ ] ensure, like in original design, the logo is on the left and text on the right. Remember, this is someone's design we are personalizing. 
- [ ] please kindly remove the partner section in the desktop version of the about page. ensure the original design is well followed. 
- [ ] Please there is this extra space below the footer of the about page desktop version. kindly remove it so the original design can be respected.

your design
![Screenshot (1158)_LI](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/4fd4388a-2720-486b-a3fa-b12211030056)
original
![Screenshot (1159)](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/64d972ad-4393-435f-9926-19990b66e7fa)

Reply:
Thank you so much for your review, I have fixed the paddings and margins around the cards.